packages/components/modules/profiles/ProfilePopover/ProfileMenuItem/index.tsx

```diff
@@ -0,0 +1,50 @@
+import { FC } from 'react'
+
+import { AvatarWithPlaceholder, CheckMarkIcon } from '@baseapp-frontend/design-system'
+
+import { Box, Typography } from '@mui/material'
+import { readInlineData } from 'react-relay'
+
+import { ProfileItemFragment$key } from '../../../../__generated__/ProfileItemFragment.graphql'
+import { ProfileItemFragment } from '../../graphql/queries/ProfileItem'
+import { StyledMenuItem } from './styled'
+import { ProfileMenuItemProps } from './types'
+
+const ProfileMenuItem: FC<ProfileMenuItemProps> = ({
+  profileRef,
+  currentProfile,
+  onProfileChange,
+  avatarProps = {},
+  width = 36,
+  height = 36,
+}) => {
+  const profile = readInlineData<ProfileItemFragment$key>(ProfileItemFragment, profileRef)
```

Comment:
why do we read from inline instead of using `useFragment`?

Reply:
Inside of the useCurrentProfile I needed it because I couldn't use the hook (useFragment) anywhere (it's a lint rule, If I'm not mistaken). So I found this alternative. I can rollback and ignore the lint rule or create two fragments, one with the inline rule and the other with the useFragment hook. What do you think would be the best approch?

Reply:
As aligned, we decided to keep it like that for this first version.